types/default-theme.d.ts

```diff
@@ -142,6 +142,7 @@ export namespace DefaultTheme {
   export interface SidebarItem {
     text: string
     link: string
+    items?: SidebarItem[]
```

Comment:
The UI looks cool. But I guess this link should be made optional if items is provided.

Reply:
This type would work:

```ts
  export type SidebarItem =
    | { text: string; link: string }
    | { text: string; link?: string; items: SidebarItem[] }
```

But not providing link is messing up some UI.